Common/Global/GlobalSOTSItem.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.ModLoader;
+using System;
+using System.Collections.Generic;
+using TheBereftSouls.Common.Utility;
+
+namespace TheBereftSouls.Common.Global
+{
+	public class GlobalSOTSItem : GlobalItem
+	{
+		//Untested - JIT used to prevent crashes if Calamity isn't enabled.
+		
+
+		// Changes following SOTS Weapons To Rogue
+		private static List<string> SOTSWeaponsToRogue = new List<string>
+		{
+			"GelAxe",
+			"WingedKnife",
+			"BerryBombs",
+			"ExplosiveKnife"
+		};
+
+		[JITWhenModsEnabled("CalamityMod","SOTS")]
+		public override void SetDefaults(Item item)
+		{
+			// Checks to see if Calamity and SOTS are installed before making changes.
+			if (!ModLoader.TryGetMod("CalamityMod", out Mod CalamityMod) | !ModLoader.TryGetMod("SOTS", out Mod SOTS))
+			{
+				return;
+			}
+			// Uses list to change all listed weapons to Rogue
+			foreach (string SOTSWeaponName in SOTSWeaponsToRogue)
+			{
+				if (item.type == SOTS.Find<ModItem>(SOTSWeaponName).Type)
+				{
+					item.DamageType = GetModClass.rogueDamageClass;
+					base.SetDefaults(item);
```

Comment:
Same as [this comment](https://github.com/bromeex/TheBereftSouls/pull/9/files#r1793423317 "#9 Common/Global/GlobalVanillaItem.cs Line 39 commit: eef2c06d9cbb21c3fb344de9393cc472519491b0 (\"Test Commit
\")").